tests/Client/SymfonyClientTest.php

```diff
@@ -187,11 +192,22 @@ public function queryProvider(): iterable
             'query' => static function (): SelectQuery {
                 return SelectQuery::create()
                     ->query('*:*')
-                    ->filter([(string) Collapse::create('sample_int')])
+                    ->filter([(string) Collapse::create('sample_int')->cache(false)])
```

Comment:
itt miért is kellett configolni a cache-t?

Reply:
Nem volt muszáj, csak bekerült a Collapse-ba is a cache és gondoltam beleteszem a tesztbe is.